lib/spack/spack/package.py

```diff
@@ -2,7 +2,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-# flake8: noqa: F401
+# flake8: noqa: F401, E402
```

Comment:
Why was this added?

Reply:
oops -- this is from by branch for `ruff` support.